pkg/contracts/src/RegistryCommunity.sol

```diff
@@ -204,7 +212,7 @@ contract RegistryCommunity is ReentrancyGuard, AccessControl {
         if (params._registerStakeAmount == 0) {
             revert ValueCannotBeZero();
         }
-        registerStakeAmount = params._registerStakeAmount; //@todo can be zero?
+        registerStakeAmount = params._registerStakeAmount;
```

Comment:
that question still without answer we want allow that be zero or enforce it to no be?

Reply:
We can allow, but then no points for fixed systems